src/v3/src/util/formUtils.ts

```diff
@@ -390,3 +396,102 @@ export const shouldHideIdentifier = (
 
   return false;
 };
+
+const getPhoneVerificationSubtitleTextContent = (
+  step: PhoneVerificationStep,
+  primaryMethod: PhoneVerificationMethodType,
+  phoneNumber?: string,
+  nickname?: string,
+): string => {
+  if (![
+    IDX_STEP.CHALLENGE_AUTHENTICATOR,
+    IDX_STEP.AUTHENTICATOR_VERIFICATION_DATA,
+  ].includes(step)
+    || !['sms', 'voice'].includes(primaryMethod)) {
+    // If step or method is invalid, return.
+    return '';
+  }
+  type PhoneVerificationI18nCondition = 'withPhoneWithNickName'
+  | 'withPhoneWithoutNickName'
+  | 'withoutPhone';
+  type I18nMapType = Record<
+  PhoneVerificationStep,
+  Record<PhoneVerificationMethodType, Record<PhoneVerificationI18nCondition, string>>
+  >;
+  const i18nMap: I18nMapType = {
+    [IDX_STEP.CHALLENGE_AUTHENTICATOR]: {
+      sms: {
+        withPhoneWithNickName: 'oie.phone.verify.sms.codeSentText.with.phone.with.nickname',
+        withPhoneWithoutNickName: 'oie.phone.verify.sms.codeSentText.with.phone.without.nickname',
+        withoutPhone: 'oie.phone.verify.sms.codeSentText.without.phone',
+      },
+      voice: {
+        withPhoneWithNickName: 'oie.phone.verify.mfa.calling.with.phone.with.nickname',
+        withPhoneWithoutNickName: 'oie.phone.verify.mfa.calling.with.phone.without.nickname',
+        withoutPhone: 'oie.phone.verify.mfa.calling.without.phone',
+      },
+    },
+    [IDX_STEP.AUTHENTICATOR_VERIFICATION_DATA]: {
+      sms: {
+        withPhoneWithNickName: 'oie.phone.verify.sms.sendText.with.phone.with.nickname',
+        withPhoneWithoutNickName: 'oie.phone.verify.sms.sendText.with.phone.without.nickname',
+        withoutPhone: 'oie.phone.verify.sms.sendText.without.phone',
+      },
+      voice: {
+        withPhoneWithNickName: 'oie.phone.verify.call.sendText.with.phone.with.nickname',
+        withPhoneWithoutNickName: 'oie.phone.verify.call.sendText.with.phone.without.nickname',
+        withoutPhone: 'oie.phone.verify.call.sendText.without.phone',
```

Comment:
Before merging: are all of these translations already translated and merged into current `master`?

Reply:
Yessir! Those were merged apart of [this](https://oktainc.atlassian.net/browse/OKTA-660939) ticket. Will wait to merge this until after this upcoming monthly to ensure the translations will be available when this is released.